src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/completion/autocomplete.ts

```diff
@@ -114,8 +138,8 @@ function inferenceEndpointToCompletionItem(
     }),
     kind: 'Reference',
     label: inferenceEndpoint.inference_id,
-    sortText: '1',
-    text: `\`${inferenceEndpoint.inference_id}\` `,
+    sortText: inferenceEndpoint.inference_id,
```

Comment:
What kind of values does the inference_id take that we can use them for sortText?

Reply:
I think it sorts alphabetically by default if the items share the same sortText